packages/cst-dts-gen/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@chevrotain/cst-dts-gen",
+  "version": "8.0.1",
+  "description": "Generates type definitions for chevrotain CST nodes",
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/Chevrotain/chevrotain/issues"
+  },
+  "license": "Apache-2.0",
+  "typings": "api.d.ts",
+  "main": "lib/src/api.js",
+  "files": [
+    "api.d.ts",
+    "lib/src/**/*.js",
+    "lib/src/**/*.js.map"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Chevrotain/chevrotain.git"
+  },
+  "scripts": {
+    "---------- CI FLOWS --------": "",
+    "build": "npm-run-all clean compile",
+    "test": "npm-run-all coverage",
+    "---------- DEV FLOWS --------": "",
+    "update-snapshots": "node ./scripts/update-snapshots.js",
+    "---------- BUILD STEPS --------": "",
+    "clean": "shx rm -rf dev",
+    "compile:watch": "tsc -w",
+    "compile": "tsc",
+    "coverage": "nyc mocha"
+  },
+  "dependencies": {
+    "chevrotain": "9.0.1",
+    "@chevrotain/utils": "9.0.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.15",
```

Comment:
don't we have a dev-dep to `@types/chai" in the root package.json?

Reply:
indeed there is 🤦